### ISSUE TITLE
"undefined undefined" when opening .graphql file containing queries

### ISSUE THREAD

**Issue by marktani** [prismagraphql/vscode-graphql#15]
opening `graphql-server-example/queries/booking.graphql` from https://github.com/graphcool/graphql-server-example produces a lot of error output:

```
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'src/generated/prisma.graphql',
     includes: [ 'prisma.graphql', 'seed.graphql', 'datamodel.graphql' ],
     extensions: 
      { prisma: 'prisma/prisma.yml',
        codegen: [Object],
        endpoints: [Object],
        customDirectives: [Object] } },
  configPath: '/Users/marktani/projects/playground/prisma/triaging/graphql-server-example/.graphqlconfig.yml',
  projectName: 'prisma' } /Users/marktani/projects/playground/prisma/triaging/graphql-server-example/src/generated/prisma.graphql
Definition not found for GraphQL type DateTimeDefinition not found for GraphQL type StringDefinition not found for GraphQL type StringDefinition not found for GraphQL type StringDefinition not found for GraphQL type StringDefinition not found for GraphQL type BooleanDefinition not found for GraphQL type StringDefinition not found for GraphQL type StringDefinition not found for GraphQL type BooleanZEBRA 1: undefined undefined
(node:46935) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property 'schemaPath' of undefined
(node:46935) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[Error - 11:48:31 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:48:32 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:48:33 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:49:55 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:49:57 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:49:57 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
[Error - 11:50:01 AM] Request textDocument/hover failed.
  Message: Request textDocument/hover failed with message: Cannot read property 'projectName' of undefined
  Code: -32603 
```

**Comment by finneganh** [comment]
Seeing this on the 0.0.8 I got from the marketplace.

```
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: './services-js/commissions-app/graphql/schema.graphql',
     includes: [ './services-js/commissions-app/src/client/**/*.{graphql,ts,tsx,js,jsx}' ],
     extensions: { endpoints: [Object] } },
  configPath: '/Users/FinneganHopkins/Source/monorepo/.graphqlconfig',
  projectName: 'commissions-app' } /Users/FinneganHopkins/Source/monorepo/services-js/commissions-app/graphql/schema.graphql
```

Query file (`./services-js/commissions-app/src/client/graphql/test.graphql`):
```
query {
  commissions {
    applyUrl
  }
}
```

**Comment by jhalborg** [comment]
Reproduction:

1. Create a `create-react-app` with TypeScript (`create-react-app my-app --scripts-version=react-scripts-ts` from https://github.com/wmonk/create-react-app-typescript)
2. Add `graphql-tag` and `graphql` to project
3. Install the plugin to VSCode
4. Clone an exisiting schema (or fetch your own) into `schema.graphql` in project root
5. Go into `App.tsx` and add

```
import gql from 'graphql-tag';

const TEST_QUERY = gql`
  query {
    // Expect automagical Intellisense and validation, be disappointed :)
  }
`
```
6. Check the "Output" pane in VS Code for `GraphQL Language Server`, see 
```
ZEBRA 1: undefined undefined
ZEBRA: GraphQLProjectConfig {
  config: 
   { schemaPath: 'schema.graphql',
     includes: [ 'schema.graphql' ],
     extensions: { endpoints: [Object] } },
  configPath: '[path to project]/.graphqlconfig.yaml',
  projectName: 'app' } [path to project]/schema.graphql
```

**Comment by divyenduz** [comment]
@marktani  The original problem of vague logs in output "GraphQL Language Server" are resolved in `0.0.9`. 

The issue with autocomplete can be logged [following these instructions](https://github.com/prismagraphql/vscode-graphql#b-testinglogging-typescript-graphql-plugin-features) and is unrelated to output "GraphQL Language Server" logs. 

I will investigate it further under this issue (https://github.com/prismagraphql/vscode-graphql/issues/21). Closing this one.
